src/pages/Login/IdPwInput.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class IdPwInput extends Component {
+  render() {
+    return (
```

Comment:
```suggestion
class IdPwInput extends Component {
  const { input, onchanging } = this.props;
  render() {
    return (
```
구조분해할당을 사용하면 가독성 좋게 사용할 수 있습니다!